### DISCUSSION TITLE
Geometric Adstock: Understanding the Output plots

### DISCUSSION THREAD

**Discussion by jamesvrt** [facebookexperimental/Robyn#597] [General]
I'm trying to understand the output plots describing geometric adstock. As I understand it, these two plots below should align, given that a high theta produces a high carryover. However, there is no clear alignment between these plots. Additionally, some vendors have 0% reported for both carryover and immediate, which looks like a bug.

Could someone help me understand what these plots represent if I have misunderstood?

I've removed the channel names due to confidentiality, but their plot order is the same.

![image](https://user-images.githubusercontent.com/29928196/211031377-85c3b5cc-3c2d-46c5-97d6-26cfed3ed1ca.png)

**Comment by AdimDrewnik** [comment]
I noticed the same problem and found no answers.
